README_new.md

```diff
@@ -0,0 +1,132 @@
+
+We also aim to be as flexible and inclusive as possible, equally accepting applications from candidates who want to work on-site at our office in London, or who want to work remotely, or who have a limited amount of time to write a demo project, or who have open-source contributions.
+
+### Our expectations
+
+babylon aims to hire only the most qualified people available at any given time. That doesn't mean that a candidate must have several academic degrees or have years and years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently** and to **engage**. As effective as they may be, we don't want robots.
+
+We also want to hire people who are **reasonable**, **professional**, and **fun** to work with. After all, they are people we will spend a great deal of time interacting with on a daily basis.
+
+As such, these are the main areas the iOS team will focus our assessment of candidates on:
+
+- Technical expertise
+ - Languages (Swift, Objective-C) and platforms (iOS, watchOS, tvOS, macOS)
+ - Architectural design patterns (MVC, MVVM, Viper, etc)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
+ - Knowledge of reactive programming
+ - Interview demo project
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+### The iOS recruitment process
+
+Our recruitment process is different between candidates who are interested in working remotely and candidates who are interested in working on-site at our office in London, in **two** ways:
+- remote candidates are *required* to be proficient in **reactive programming** (`Reactive Swift`/`Reactive Cocoa` or `RxSwift`) at an above-average level
+- remote candidates go through a longer technical interview process:
+    * on-site: initial call -> demo -> 3-hour face-to-face technical interview
+    * remote: initial call -> demo -> one or more Skype interviews -> 1 working day with us (at our expense)
+
+As indicated above, we expect all candidates to submit an "interview demo project" but, in order to be flexible, we provide 3 different ways in which a candidate can submit a demo of their work. **Only one of these is required**.
+
+1. [Do the babylon demo project](TODO)
+2. [Send us a project you've already done](TODO)
+3. [Show us some open source work you did](TODO)
+
+A demo project should be an example of one's **best** work. We don't just want to see a technically good project; we want to be *wowed* by a candidate's techincal skills and creativity.
```

Comment:
To be honest I find this a bit over the top. I get the reasoning behind this, but this is to be expected.

I'd propose to remove this, but if we keep it, maybe having something like `A demo project should be an example of one's **best** work, something that the candidate is proud of.`

Reply:
I honestly don't see why it's over the top. Also, a candidate could be proud of their work even if the work is bland. I'm sure that Leo was proud of his demo but we thought it lacked a "soul".